test/gurka/gurka.cc

```diff
@@ -571,11 +571,20 @@ findEdgeByNodes(valhalla::baldr::GraphReader& reader,
                            end_node_name);
 }
 
+std::string
+do_action(const map& map, valhalla::Api& api, std::shared_ptr<valhalla::baldr::GraphReader> reader) {
```

Comment:
this is the way to test the pbf interface in gurka, basically an `actor::act` wrapper